### DISCUSSION TITLE
DLC/ Multiple Animal Tracking/Data Swapping Between Animals

### DISCUSSION THREAD

**Discussion by Somoday** [AdaptiveMotorControlLab/CEBRA#158] [Q&A] [answered]
Hi everyone,

I am reaching out to seek your help in understanding a problem with my project on multiple animal tracking. I am using DLC 2.2.3, and my project involves tracking multiple animals. The issue I am encountering is that after analyzing the videos, I find inconsistencies in the CSV files. Specifically, when I recheck the CSV files and compare them with the video, I find that the tracking data of one animal is swapped with another. This happens in most of my videos.

![DLC](https://github.com/user-attachments/assets/55ef4487-9c36-4159-a7c1-2baf3446dc82)

What could be the solution to this problem?

Thank you in advance.

**Comment by MMathisLab** [comment, marked as answer]
I think you meant to post this to the DeepLabCut project, not CEBRA (we don't support pose estimation models, only the outputs). I would recommend posting on the image forum: https://forum.image.sc/tag/deeplabcut

**Reply by Somoday** [reply]
thank you
